test/client/fixtures/sandbox/event/internal-listeners-test.js

```diff
@@ -157,6 +157,32 @@ test('initElementListening', function () {
     checkHandlerCounters(1, 2, 2, 1);
 });
 
+test('getEventListeners', function () {
+    var onClick        = function () {
+    };
+
+    strictEqual(listeners.getEventListeners(container, 'click'), null);
```

Comment:
For consistency use the `strictEqual` for `null` check and `deepEqual` with array check.